File: Celciuc_to_fahrenheit.java
/*   In java Celciuc to Fahrenheit */
import java.util.Scanner;
public class Program {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		System.out.println("enter temp ::: ");
		float celc = scanner.nextFloat();
		float fahren = celc*9/5+32 ;
    System.out.println("in fahrenheit ::: " + fahren);
	}
}

File: Celciuc_to_fahrenheit.c
/*    In C Celciuc to Fahrenheit  */
#include<stdio.h>
int main()
{
    float celc, fahren;
    printf("enter temp ::: ");
    scanf("%f",&celc);
    fahren = celc*9/5+32;
    printf("in fahrenheit ::: %.2f",fahren);
    return 0;
}
